specification.markdown

```diff
@@ -1121,29 +1121,31 @@ Element IDs are described in section Element ID. Element IDs are encoded using t
 
 This IANA Registry only applies to Elements that can be contained in the EBML Header, thus including Global Elements. Elements only found in the EBML Body have their own set of independent Element IDs and are not part of this IANA Registry.
 
-The VINT Data value of one-octet Element IDs MUST be between 0x01 and 0x7E. These items are valuable because they are short, and need to be used for commonly repeated elements. Values from 1 to 126 are to be allocated according to the "RFC Required" policy [@!RFC8126].
+One-octet Element IDs MUST be between 0x81 and 0xFE. These items are valuable because they are short, and need to be used for commonly repeated elements. Element IDs are to be allocated within this range according to the "RFC Required" policy [@!RFC8126].
 
-The VINT Data value of two-octet Element IDs MUST be between 0x007F and 0x3FFE. Numbers are to be allocated within this range according to the "Specification Required" policy [@!RFC8126].
+The following one-octet Element ID is RESERVED: 0x80.
```

Comment:
Shouldn't this also list 0xFF? The second-to-last line in Table 3 would seem to put it in a special category (like the other all-consecutive-one bit patterns listed in the paragraphs below).